src/esrs/postgres/builder.rs

```diff
@@ -84,17 +86,14 @@ where
         self
     }
 
-    /// This function sets up the database in a transaction and returns an instance of PgStore.
-    ///
-    /// It will create the event store table (if it doesn't exist) and two indexes (if they don't exist).
-    /// The first one is over the `aggregate_id` field to speed up `by_aggregate_id` query.
-    /// The second one is a unique constraint over the tuple `(aggregate_id, sequence_number)` to avoid race conditions.
+    /// This function runs all the needed [`Migrations`], transactional set up the database.
```

Comment:
```suggestion
    /// This function runs all the needed [`Migrations`], transactionally setting up the database.
```

Or something like that

Reply:
Do the word `transactionally` actually exists?

Reply:
I think it does, but I would maybe use the word atomically here?